mcs_benchmark_data/_model.py

```diff
@@ -26,7 +26,7 @@ def _properties_to_rdf(
             else:
                 resource.add(property_definition.uri, Literal(property_.value))
 
-    def to_rdf(self, *, graph: Graph, property_definitions) -> Resource:
+    def to_rdf(self, *, graph: Graph) -> Resource:

```

Comment:
In the body of this, self.uri is declared as URIRef, so can't be None. In paradicms is use self.uri = None to indicate a _Model's Resource has a BNode identifier. You can do that if you want, makes things a little cleaner, but the need may not be there.